src/cmd/run.go

```diff
@@ -364,8 +364,6 @@ func runCommandWithFallbacks(container string, command []string, emitEscapeSeque
 			return nil
 		}
 	}
-
-	panic("code should not be reached")
```

Comment:
The loop before this statement is an infinite loop without a break statement. This statement is never executed.

Reply:
Yes, that's why it's a self-documenting assertion - it clearly says that it shouldn't be reached.  Otherwise, our code would have been panicking.  :)

The infinite loop without any final `return` statement makes the function somewhat unusual, and the code isn't entirely trivial.  We have had [bugs in this code](https://github.com/containers/toolbox/commit/d6b1fbea4cf29bd6c826020d3f957054a57d65a8) that would prevent the user from going out of a `toolbox enter`.

We could have also used a comment, but a comment won't complain as loudly when things go wrong.

Reply:
In the end, I went for a comment.  See https://github.com/containers/toolbox/pull/1185 for the full explanation.